packages/terra-slider/src/Slider.module.scss

```diff
@@ -0,0 +1,212 @@
+@import '~terra-mixins/lib/Mixins';
+
+// Themes
+@import './orion-fusion-theme/Slider.module';
+@import './clinical-lowlight-theme/Slider.module';
+
+:local {
+  .slider {
+    align-items: center;
+    display: inline-flex;
+    min-height: 44px;
+  }
+
+  .label {
+    color: var(--terra-slider-label-color, #1c1f21);
+    font-weight: 700;
+    margin-right: 7px;
```

Comment:
You should have theme variables from the font weight and margin

Reply:
These values are same in all themes, I don't see a need to have theme variables for these.

Reply:
These are not the only themes.  We have the Redwood theme coming up as well.  There is also the cerner-clinical-theme.  We cannot limit to just designing for lowlight and fusion.

Reply:
Updated.